Source/WTF/wtf/ThreadAssertions.h

```diff
@@ -58,8 +58,8 @@ class ThreadLike {
 public:
     // Never returns 0.
     WTF_EXPORT_PRIVATE static uint32_t currentSequence();
-
 protected:
+    static constexpr uint32_t s_mainThreadID { 1 };
```

Comment:
I think typical constexpr might be named without s_ prefix.. (like, it's just a constant)

Reply:
I hesitated on that, thanks for confirming.